src/tests/mock-server/config/commands.groovy

```diff
@@ -0,0 +1,50 @@
+import groovy.json.JsonOutput
+import java.time.Instant
+
+def generateUUIDv7() {
+    def now = Instant.now().toEpochMilli()
+    def timeHex = String.format("%012x", now)
+    def randomBits = UUID.randomUUID().toString().replace("-", "").substring(12)
+    return "${timeHex.substring(0, 8)}-${timeHex.substring(8)}-7${randomBits.substring(0, 3)}-" +
+       "${randomBits.substring(3, 7)}-${randomBits.substring(7)}"
+
+}
+
+def actions = [
+    ["name": "set-group", "version": "v5.0.0", "args": ["validYaml", "invalidYaml"]],
+    ["name": "set-group", "version": "v5.0.0", "args": []],
+    ["name": "set-group", "version": "v5.0.0", "args": [""]],
+    ["name": "set-group", "version": "v5.0.0", "args": ["validYaml", 8]],
+    ["name": "set-group", "version": "v5.0.0", "args": ["validYaml"]],
+    ["name": "update-group", "version": "v5.0.0", "args": ["noNeedArgs"]],
+    ["name": "update-group", "version": "v5.0.0"],
+    ["name": "update-group", "version": "v5.0.0", "args": ""]
+]
+
+def numCommands = new Random().nextInt(3)
```

Comment:
I was testing this branch together with one I'm working on and after setting `numCommands` to a 1000 I got some segmentation faults. Then I checked only in this branch and it also happens. Looks like is not related to the branch though, but just exposed by it. I want to make sure that setting this variable to a high value is valid and that my test is ok, what do you think?